app/test_split_integer.py

```diff
@@ -2,20 +2,27 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    goals = split_integer(32, 6)
+    assert sum(goals) == 32, "sum of the parts should be equal value"
 
 
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+def test_should_split_into_equal_parts_divisible_by_parts() -> None:
+    goals = split_integer(16, 4)
+    assert goals == [4, 4, 4, 4], "should split into equal parts"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    goals = split_integer(8, 1)
```

Comment:
According to the checklist, avoid creating unnecessary variables for simple tests. You can directly use `assert split_integer(8, 1) == [8]`.